src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -98,20 +98,22 @@ void GcsServer::Start() {
   rpc_server_.RegisterService(*worker_info_service_);
 
   auto load_completed_count = std::make_shared<int>(0);
-  int load_count = 3;
+  int load_count = 2;
   auto on_done = [this, load_count, load_completed_count]() {
     ++(*load_completed_count);
 
     if (*load_completed_count == load_count) {
-      // Start RPC server when all tables have finished loading initial data.
-      rpc_server_.Run();
-
-      // Store gcs rpc server address in redis.
-      StoreGcsServerAddressInRedis();
-      is_started_ = true;
+      auto on_done = [this]() {
```

Comment:
```suggestion
      auto actor_manager_load_initial_data_callback = [this]() {
```
Avoid using the same var name. And document that actor data needs be to loaded lastly.

Reply:
fixed